js/main.js

```diff
@@ -44,14 +44,27 @@ document.addEventListener("DOMContentLoaded", function(event) {
         }
       });
 
-      var owl = $('.solstice-slider');
-      owl.owlCarousel({
-          items:1,
-          autoplay:true,
-          autoplayTimeout:6000,
-          autoplayHoverPause:true,
-          autoplaySpeed: 2000,
-          loop:true,
+      const solsticeSliderHome = () => {
+        var owl = $('.solstice-slider');
+        owl.owlCarousel({
+            items:1,
+            autoplay:true,
+            autoplayTimeout:6000,
+            autoplayHoverPause:true,
+            autoplaySpeed: 2000,
+            loop:true,
+        });
+      }
+      
+      $(window).on("load", function() {
+          solsticeSliderHome();
+      });
+      
+      $("body").on("shown.ef.featured_story", function(e) {
```

Comment:
What is the purpose of this? 

I feel this we are doing something wrong when I see owl.trigger('destroy.owl.carousel'); then  owl.trigger('refresh.owl.carousel');

Reply:
its to make sure the carousel is loaded after the content is added. This is also something we are doing on ECDtools

Reply:
I feel this is the way we should do it:
https://www.jq22.com/demo/OwlCarousel2/demos/ajax.html

There is an event that you can trigger for inserting content.

Reply:
turns out I was using the same class for 2 different carousel on the page which was causing conflicts.

I update the class name for the Featured Story slider and removed the unnecessary code and both sliders are working well.